src/utils/storage.ts

```diff
@@ -0,0 +1,83 @@
+import { ImporterState } from '../importer/types';
+import { SheetDefinition } from '../sheet/types';
+
+const DB_NAME = 'HelloCSV';
```

Comment:
How does it work when app integrates the plugin twice - like renders 2 different importers? Does each importer have its own db?

Reply:
no this should have the same database, just different key,values in said database

Reply:
in the indexDBConfig, they could add a custom key prefix if needed. (if they happen to have two importers with the exact sheetDefinitions